### ISSUE TITLE
Quiet Build Logs

### ISSUE THREAD

**Issue by ralph-burstsms** [JulianMar/nuxt-bugsnag#42]
Can we turn off bugsnag build logs, or atleast the uploading sourcemap part on application build? e.g.

```
10:42:54 AM: ℹ Enabling uploading of release sourcemaps to Bugsnag
10:42:54 AM:  WARN  devtool has been normalized to source-map as webpack documented value
10:42:54 AM: ℹ Enabling uploading of release sourcemaps to Bugsnag
10:43:42 AM: <i> [BugsnagSourceMapUploaderPlugin] uploading sourcemap for "*/9267c19.js"
10:43:42 AM: <i> [BugsnagSourceMapUploaderPlugin] uploading sourcemap for "*/e8c6e4d.js"
10:43:42 AM: <i> [BugsnagSourceMapUploaderPlugin] uploading sourcemap for "*/c687de7.js"
10:43:42 AM: <i> [BugsnagSourceMapUploaderPlugin] uploading sourcemap for "*/cb22680.js"
10:43:42 AM: <i> [BugsnagSourceMapUploaderPlugin] uploading sourcemap for "*/826bee0.js"
10:43:42 AM: <i> [BugsnagSourceMapUploaderPlugin] uploading sourcemap for "*/db20914.js"
10:43:42 AM: <i> [BugsnagSourceMapUploaderPlugin] uploading sourcemap for "*/a52a43d.js"
10:43:42 AM: <i> [BugsnagSourceMapUploaderPlugin] uploading sourcemap for "*/528c497.js"
10:43:42 AM: <i> [BugsnagSourceMapUploaderPlugin] uploading sourcemap for "*/9d1b6cf.js"
10:43:42 AM: <i> [BugsnagSourceMapUploaderPlugin] uploading sourcemap for "*/46f82b4.js"
10:43:42 AM: <i> [BugsnagSourceMapUploaderPlugin] uploading sourcemap for "*/3eaee49.js"
10:43:42 AM: <i> [BugsnagSourceMapUploaderPlugin] uploading sourcemap for "*/9eb38b9.js"
10:43:42 AM: <i> [BugsnagSourceMapUploaderPlugin] uploading sourcemap for "*/a7af11b.js"
10:43:42 AM: <i> [BugsnagSourceMapUploaderPlugin] uploading sourcemap for "*/e172e82.js"
10:43:42 AM: <i> [BugsnagSourceMapUploaderPlugin] uploading sourcemap for "*/52e23d0.js"
10:43:42 AM: <i> [BugsnagSourceMapUploaderPlugin] uploading sourcemap for "*/2f6c77c.js"
10:43:42 AM: <i> [BugsnagSourceMapUploaderPlugin] uploading sourcemap for "*/9b4d136.js"
10:43:42 AM: <i> [BugsnagSourceMapUploaderPlugin] uploading sourcemap for "*/b03e51b.js"
10:43:42 AM: <i> [BugsnagSourceMapUploaderPlugin] uploading sourcemap for "*/7c3ae19.js"
```

**Comment by ralph-burstsms** [comment]
Hi @JulianMar , I'm okay with publishing sourcemaps, i just want these logs to be silent. Is there a way for that?

**Comment by JulianMar** [comment]
Hey, this got fixed in V6.1 
check the docs for disabling it.
